health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/NewDependentFieldWrapper.js

```
@@ -1032,277 +1032,828 @@ function NewDependentFieldWrapper({ t }) {
+                                                          removeSubCondition(
+                                                            idx
+                                                          )
+                                                        }
+                                                        title={
+                                                          removeConditionLabel
+                                                        }
+                                                        aria-label={
+                                                          removeConditionLabel
+                                                        }
+                                                        role="button"
+                                                      >
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)